github_api/voting.py

```diff
@@ -118,15 +118,8 @@ def get_vote_weight(api, username):
     created = arrow.get(user["created_at"])
     age = (now - created).total_seconds()
     old_enough_to_vote = age >= settings.MIN_VOTER_AGE
-    age_weight = 1.0 if old_enough_to_vote else 0.0
+    weight = 1.0 if old_enough_to_vote else 0.0
```

Comment:
Why not just `return float(age >= settings.MIN_VOTER_AGE)`?  Python guarantees that `True` and `False` always convert to `1.0` and `0.0`.